### PR TITLE
BXMSDOC-4454 Integrating Red Hat Business Automation with Red Hat Single Sign-On for DM/PAM 7.5

### PR DESCRIPTION
**Pull request by emmurphy1**
See jira for details;
[BXMSDOC-4454](https://issues.jboss.org/browse/BXMSDOC-4454)

### CODE REVIEW

File: doc-content/enterprise-only/integration/sso-kie-server-client-adapter-proc.adoc [hunk 1]
```diff
@@ -21,11 +21,11 @@ If you deployed {KIE_SERVER} to a different application server than {CENTRAL}, i
 * *Version:* {RH-SSO_VERSION}
 . Download *Red Hat Single Sign-on {RH-SSO_VERSION_LONG} Client Adapter for JBoss EAP 7* (`rh-sso-{RH-SSO_VERSION_LONG}-eap7-adapter.zip`).
 . Unzip and install `rh-sso-{RH-SSO_VERSION_LONG}-eap7-adapter.zip`. For installation instructions, see the "JBoss EAP Adapter" section of the https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/{RH-SSO_VERSION}/html-single/securing_applications_and_services_guide/#jboss_adapter[_Red Hat Single Sign On Securing Applications and Services Guide_].
```
Comment (mramendi) [inline doc-content/enterprise-only/integration/sso-kie-server-client-adapter-proc.adoc]: is "unzip" ok style guide wise?

File: doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc [hunk 2]
```diff
@@ -15,13 +15,33 @@ After you install RH-SSO, you must install the RH-SSO client adapter for {EAP} a
 * *Version:* {RH-SSO_VERSION}
 . Download *Red Hat Single Sign-on {RH-SSO_VERSION} Client Adapter for JBoss EAP 7* (`rh-sso-{RH-SSO_VERSION}-eap7-adapter.zip`).
 . Unzip and install `rh-sso-{RH-SSO_VERSION}-eap7-adapter.zip`. For installation instructions, see the "JBoss EAP Adapter" section of the https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.3/html-single/securing_applications_and_services_guide/#jboss_adapter[_Red Hat Single Sign On Securing Applications and Services Guide_].
-. Go to `_EAP_HOME_/standalone/configuration` and open the `standalone.xml` and `standalone-full.xml` files. 
++
+NOTE: Install the adapter with the `-Dserver.config=standalone-full.xml` property.
```
[the remark this answers is not in the document]
Comment (emmurphy1) [reply doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc:19]: Tomas, can you clarify, please?

File: doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc [hunk 3]
```diff
@@ -15,13 +15,33 @@ After you install RH-SSO, you must install the RH-SSO client adapter for {EAP} a
 * *Version:* {RH-SSO_VERSION}
 . Download *Red Hat Single Sign-on {RH-SSO_VERSION} Client Adapter for JBoss EAP 7* (`rh-sso-{RH-SSO_VERSION}-eap7-adapter.zip`).
 . Unzip and install `rh-sso-{RH-SSO_VERSION}-eap7-adapter.zip`. For installation instructions, see the "JBoss EAP Adapter" section of the https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.3/html-single/securing_applications_and_services_guide/#jboss_adapter[_Red Hat Single Sign On Securing Applications and Services Guide_].
```
[the remark this answers is not in the document]
Comment (emmurphy1) [reply doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc]: Thank you. The link should not have been to the section, so I removed the anchor.

File: doc-content/enterprise-only/integration/sso-kie-server-client-adapter-proc.adoc (hunk 1 above)
Comment (emmurphy1) [reply doc-content/enterprise-only/integration/sso-kie-server-client-adapter-proc.adoc]: No, I changed to 'extract'.

File: doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (tomasdavidorg) [reply doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc:19]: It is during the installation of the adapter. So if you take a look into the SSO documentation there is the command:

`$ ./bin/jboss-cli.sh --file=bin/adapter-elytron-install-offline.cli` 

and with the property is should be

`$ ./bin/jboss-cli.sh --file=bin/adapter-elytron-install-offline.cli -Dserver.config=standalone-full.xml`

**Pull request merged**